netsim/ansible/templates/gateway/frr.j2

```diff
@@ -13,8 +13,9 @@ if [ ! -e /sys/class/net/{{ v_if }} ]; then
   ip link set dev {{ v_if }} address {{ v_mac }} addrgenmode {{ 'none' if afm == 'ipv4' else 'random' }}
   ip addr add {{ intf.gateway[afm] }} dev {{ v_if }}
 {%     if afm=='ipv4' %}
-  sysctl -w net.ipv4.conf.{{ intf.ifname }}.arp_announce=1
+  sysctl -w net.ipv4.conf.{{ intf.ifname }}.arp_announce=2
```

Comment:
PR is merged, but this would have been a good time to create a well documented macro for these tweaks